infra/base-images/base-builder/compile_honggfuzz

```diff
@@ -20,7 +20,8 @@ export BUILD_OSSFUZZ_STATIC=true
 
 pushd $SRC/honggfuzz > /dev/null
 make clean
-CC=clang CFLAGS= make
+CC=clang CFLAGS="-O3 -funroll-loops" make
```

Comment:
Can you add a comment why these CFLAGS are used.

Reply:
Done!